src/Aspire.Hosting/ApplicationModel/ContainerPullPolicyAnnotation.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// Image pull policies for container resources.
+/// </summary>
+public enum PullPolicy
```

Comment:
Should we add a `Default = 0` enum value?

I've hoped that we would periodically pull new images (say once a day) in the future. If we don't put a 0 enum value now,  `Always` will be the 0 enum value.

Reply:
Adding a default makes sense to make future API/behavior changes more straightforward.

Reply:
Should we consider naming it `ContainerImagePullPolicy` etc, think the `PullPolicy` is a little generic

Reply:
I updated the enum and annotation to `ImagePullPolicy` and `ContainerImagePullPolicyAnnotation` respectively.